packages/expo-background-task/android/build.gradle

```
@@ -1,15 +1,27 @@
 apply plugin: 'com.android.library'
 
+def useLegacyExpoModulesCorePlugin = {
+  def expoModulesCorePlugin = new File(project(":expo-modules-core").projectDir.absolutePath, "ExpoModulesCorePlugin.gradle")
+  apply from: expoModulesCorePlugin
+  applyKotlinExpoModulesCorePlugin()
+  useCoreDependencies()
+  useDefaultAndroidSdkVersions()
+  useExpoPublishing()
+}
+
+try {
+  apply plugin: 'expo-module-gradle-plugin'
+} catch (e) {
+  if (!e instanceof UnknownPluginException) {
+    throw e
+  }
+
+  useLegacyExpoModulesCorePlugin()
+}
+
```

```suggestion
apply plugin: 'expo-module-gradle-plugin'

```
We don't need to support the legacy configuration if we don't release it in SDK 52.

---

I think it is supposed to be released in sdk-52 - just as a "hidden" feature so that we can start testing it.

---

ok, make sens 👍

---

Thanks :)